### PR TITLE
Feature implement digest email

### PR DESCRIPTION
**Pull request by KarenVentura**
@anymoto @FerPerales

### CODE REVIEW

File: app/views/digest_mailer/digest_mailer.html.erb [hunk 1]
```diff
@@ -13,11 +13,11 @@
             </td>
           </tr>
           <tr>
-          <h1 style="font-family: HelveticaNeue-Bold; font-size: 21px; color: red; line-height: 26px; margin-left: 20px;"> Recent Post </h1><br>
+          <h1 style="font-family: HelveticaNeue-Bold; font-size: 21px; color: red; line-height: 26px; margin-left: 20px;"> Recent Posts </h1><br>
             <% @posts.each do | post | %>
               <h1 style="font-family: HelveticaNeue-Bold; font-size: 21px; color: #331A48; line-height: 26px; margin-left: 20px;"><%= post.title %></h1><br>
               <div style="font-family: HelveticaNeue; font-size: 20px;color: #6E4995; line-height: 18px; padding: 15px;">
-                <p><%= post.body.html_safe %></p><br>
+                <p><%= post.number_of_comments %> comments, <%= post.votes.count %> votes, created at <%= post.created_at.strftime('%Y-%m-%d') %></p><br>
```
Comment (anymoto) [inline app/views/digest_mailer/digest_mailer.html.erb]: What if there is just one comment? "1 comments"

Please check this out http://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize

File: spec/mailers/digest_mailer_spec.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@
 describe DigestMailer do
   describe "#digest_mailer" do
     let(:user) { FactoryGirl.create(:user) }
+    let(:post) { FactoryGirl.create(:external_post, created_at: Date.today) }
```
Comment (anymoto) [inline spec/mailers/digest_mailer_spec.rb]: Do we need this?

**Comment by KarenVentura** [comment]
we finished :)

**Comment by KarenVentura** [comment]
@anymoto  @FerPerales <3

**Comment by anymoto** [comment]
LGTM :heavy_check_mark:

**Pull request merged**